server/src/main/java/io/littlehorse/common/model/metadatacommand/subcommand/PutPrincipalRequestModel.java

```diff
@@ -84,6 +84,11 @@ public Principal process(MetadataCommandExecution context) {
                 context.service().getPrincipal(context.authorization().principalId());
         PrincipalModel toSave = new PrincipalModel();
         toSave.setId(new PrincipalIdModel(id));
+
+        // Check if the ID contains a slash
+        if (id.contains("/")) {
```

Comment:
Can we also add `\` as well? We _might_ want to use that character for nested principals, which is a feature floating around in my head.

Reply:
Updated the code. 
In Linux, when we run command (with single backslash `littlehorse\1`) like 
`lhctl put principal littlehorse\1 --acl "acl_principal:write,acl_workflow:read,acl_tenant:write" --tenantId A` It sends principal id as `littlehorse1` automatically by removing the backslash (So it passes validation check). 

This command can be used instead to receive validation error from Linux command line (with double backslash `littlehorse\\1`)

`lhctl put principal littlehorse\\1 --acl "acl_principal:write,acl_workflow:read,acl_tenant:write" --tenantId A`